src/renderer/dx/CustomTextLayout.h

```diff
@@ -26,6 +26,8 @@ namespace Microsoft::Console::Render
                          const std::basic_string_view<::Microsoft::Console::Render::Cluster> clusters,
                          size_t const width);
 
+        void SetData(std::basic_string_view<Cluster> const clustor);

```

Comment:
```suggestion
        void SetData(std::basic_string_view<Cluster> const clusters);
```

typo and didn't match cpp name